components/base/FooterCTA.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Button from "./Button";
+import { PURPLE_DARK } from "../colors";
+import { H2, SubHeader } from "./Typography";
+import { maxWidth, BREAKPOINT_IPAD, BREAKPOINT_MOBILE } from "../breakpoints";
+
+// Styles
+
+const Container = styled.div`
+  padding: 60px;
+  width: 100%;
+  background-color: ${PURPLE_DARK};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  @media (${maxWidth(BREAKPOINT_MOBILE)}) {
+    padding: 10px;
+    flex-direction: column;
+  }
+  /* Text sizes decrease in smaller screens */
+  > div > h2 {
```

Comment:
This definitely wouldn't be something to address in this ticket, but I wonder if we could update the typography components to take in a `customStyles` prop or something 🤔

Reply:
Yeah, that sounds good! I believe we want to revisit Typography later, so will add this as an action item there as well! Thanks, Connor!